Error on Windows?

I'm using Windows 8.1. When trying to use it or run the test script, I get something like:
```
Started
Error: Invalid path '\bundle.js'
    at pathToArray (c:\REDACTED\webpack-require\node_modules\memory-fs\lib\MemoryFileSystem.js:27:38)
    at MemoryFileSystem.readFileSync (c:\REDACTED\webpack-require\node_modules\memory-fs\lib\MemoryFileSystem.js:75:13)
    at c:\REDACTED\webpack-require\lib\webpackRequire.js:47:19
    at Tapable.<anonymous> (c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:193:14)
    at Tapable.emitRecords (c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:281:37)
    at Tapable.<anonymous> (c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:186:11)
    at c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:274:11
    at Tapable.applyPluginsAsync (c:\REDACTED\webpack-require\node_modules\tapable\lib\Tapable.js:60:69)
    at Tapable.afterEmit (c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:271:8)
    at Tapable.<anonymous> (c:\REDACTED\webpack-require\node_modules\webpack\lib\Compiler.js:266:14)
F

Failures:
1) webpackRequire works
  Message:
    Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
  Stack:
    Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
        at Timer.listOnTimeout (timers.js:119:15)

1 spec, 1 failure
Finished in 5.006 seconds
npm ERR! Test failed.  See above for more details.
```
Could be something else, but this was my first thought.

--- a/lib/webpackRequire.js
+++ b/lib/webpackRequire.js
@@ -44,7 +44,7 @@ function webpackRequire(config, name, cb) {
       cb(statsJson.errors, null, stats);
     }
 
-    var data = fs.readFileSync(path.join(compiler.outputPath, 'bundle.js'), 'utf8');
+    var data = fs.readFileSync('/bundle.js', 'utf8');
     var script = contextify.createScript(data);
     cb(null, function() {
       var context = contextify.createContext({});

